services/web/Dockerfile.prod

```diff
@@ -0,0 +1,56 @@
+###########
+# BUILDER #
+###########
+
+FROM python:3.11 as builder
+
+# environment variables
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+ENV APP_PATH=/usr/src/app
+
+WORKDIR $APP_PATH
+
+# install system dependencies
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends gcc
+
+# lint. TODO uncomment and fix linting errors
+# RUN pip install --upgrade pip
+# RUN pip install flake8==5.0.4
+# COPY . $APP_PATH
+# RUN flake8 --ignore=E501,F401 .
```

Comment:
this part is commented out, because there are linting errors in the Python code.
I recommend to fix them and run lint regularly, in order to maintain high quality code.